pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -48,6 +49,8 @@ def update_learning_rates(
         if opt_indices is None:
             opt_indices = []
 
+        progress: OptimizationProgress = self.trainer.fit_loop.epoch_loop.batch_loop.optim_progress
```

Comment:
same as above. Is there a way to not hardcode how many nesting levels we actually have? Custom loops could either combine two loops into one or have more levels which would make this invalid.

Reply:
For now, we could do `self.trainer.fit_loop.epoch_loop.batch_loop.increment_scheduler_ready()`